Can't install webpack-command as global module

<!--
  Issues are so 🔥

  If you remove or skip this template, you'll make the 🐼 sad and @ken_wheeler
  will appear and pile-drive the close button from a great height while making
  animal noises.

  👉🏽 Need support, advice, or help? Don't open an issue.
  Head to StackOverflow or https://gitter.im/webpack/webpack.
-->

* Operating System: macos
* Node Version: latest
* NPM Version: latest
* webpack Version: latest
* webpack-command Version: latest

<!-- Please place an x (no spaces!) in all [ ] that apply -->

This issue is for a:

- [x] **bug**
- [ ] **feature** request
- [ ] **modification** request

### Code

##### CLI Command

n/a

### Expected Behavior

Module is usable

### Actual Behavior

```console
webpack -h # Error: Cannot find module 'enhanced-resolve' 
```

### How Do We Reproduce?

```console
npm i webpack -g
npm i webpack-command -g
```

<!--
  Remove this section if not reporting a bug.

  If your webpack config is over 50 lines long, please provide a URL to a repo
  for your beefy 🍖 app that we can use to reproduce.
-->

--- a/package.json
+++ b/package.json
@@ -38,8 +38,6 @@
     "lib/"
   ],
   "peerDependencies": {
-    "enhanced-resolve": "^4.0.0",
-    "loader-utils": "^1.1.0",
     "webpack": "^4.4.0"
   },
   "dependencies": {
@@ -49,6 +47,7 @@
     "chalk": "^2.3.2",
     "debug": "^3.1.0",
     "decamelize": "^2.0.0",
+    "enhanced-resolve": "^4.0.0",
     "import-local": "^1.0.0",
     "isobject": "^3.0.1",
     "loader-utils": "^1.1.0",

